continuous-delivery/build-binaries-linux-x64.yml

```diff
@@ -1,16 +1,11 @@
 version: 0.2
 phases:
-  install:
-    runtime-versions:
-      nodejs: 10
-    commands:
-      - apt-get install -y cmake
   build:
     commands:
       - mkdir linux-x64
       - cd aws-crt-nodejs
-      - bash ./continuous-delivery/build-binaries-unix.sh
-      - cp -r dist/bin/linux-x64/* ../linux-x64
+      - builder --version=v0.6.0 build --project=aws-crt-nodejs run_tests=false
```

Comment:
Is this going to create version sync problems?  Any way they can stay together?

Reply:
good catch, that was from debugging, will fix.